VirtoCommerce.PricingModule.Web/Scripts/blades/item/item-prices.js

```diff
@@ -1,4 +1,4 @@
-﻿angular.module('virtoCommerce.pricingModule')
+angular.module('virtoCommerce.pricingModule')
```

Comment:
Please revert this file, if there are no meaningful changes (only formatting).

Reply:
it`s done.